docs/user_guide.md

```diff
@@ -149,16 +149,55 @@ You can check one or more members, then click `ACTION`, choose one role to batch
 ![browse project](img/new_remove_update_member.png)
 
 ## Replicating resources  
-Replication allows users to replicate resources(images/charts) between Harbor and non-Harbor registries in both pull or push mode. Currently, the non-Harbor registries includes Docker Hub, Docker registry, Huawei SWR, and more registries will be supported in future.  
+Replication allows users to replicate resources (images/charts) between Harbor and non-Harbor registries in both pull or push mode. 
 
-Once the system administrator has set a rule, all resources that match the defined [filter](#resource-filter) patterns will be replicated to the destination registry when the [triggering condition](#trigger-mode) is matched. Each resource will start a task to run. If the namespace does not exist on the destination registry, a new namespace will be created automatically. If it already exists and the user configured in policy has no write privilege to it, the process will fail. The member information will not be replicated.  
+Once the system administrator has set a rule, all resources that match the defined [filter](#resource-filter) patterns will be replicated to the destination registry when the [triggering condition](#trigger-mode) is matched. Each resource will start a task to run. If the namespace does not exist on the destination registry, a new namespace will be created automatically. If it already exists and the user configured in the policy has no write privilege to it, the process will fail. The member information will not be replicated.  
 
-There may be a bit of delay during replication based on the situation of the network. If replication task fails, it will be re-scheduled a few minutes later and try 3 times.  
+There may be a bit of delay during replication based on the situation of the network. If a replication task fails, it will be re-scheduled a few minutes later and try 3 times.  
 
-**Note:** Due to API changes, replication between different versions of Harbor may be broken.
+**Note:** Due to API changes, replication between different versions of Harbor might not work.
+
+### Creating replication endpoints
+
+To replicate image repositories from one instance of Harbor to another Harbor or non-Harbor registry, you first create replication endpoints.
+
+1. Go to **Registries** and click the **+ New Endpoint** button.
+
+   ![New replication endpoint](img/replication-endpoint1.png)
+1. For **Provider**, use the drop-down menu to select the type of registry to set up as a replication endpoint.
+
+   The endpoint can be another Harbor instance, or a non-Harbor registry. Currently, the following non-Harbor registries are supported:
+
+   - Ali Cloud Container Registry

```

Comment:
The order I would list these in: Docker Hub, docker registry, aws, ecr, gcp, huawei, Helm Hub